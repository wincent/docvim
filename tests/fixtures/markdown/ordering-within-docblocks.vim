""
" @mappings
"
" Stuff about mappings.
"
" @footer
"
" # License
"
" Released under the terms of the OMFGWTFBBQLMAOPL.
"
" # History
"
" ## 0.1 (not yet released)
"
" - Initial release.
"
" @plugin Example An example plug-in.
"
" In which the narrator says some things.
"
