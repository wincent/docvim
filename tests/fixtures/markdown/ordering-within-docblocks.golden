# Example

In which the narrator says some things.

Stuff about mappings.

## License

Released under the terms of the OMFGWTFBBQLMAOPL.

## History

### 0.1 (not yet released)

- Initial release.
